apis/intermediate_api/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="intermediate_api",
-    version="20.8.2",
+    version="20.8.3",
```

Comment:
Is it also worth to change the core_api version as the core_api endpoints have changed too?

Reply:
I think ultimately, it is worth changing it and that 20.8. x, this x position is for minor changes.
However, the intermediate API, I can safely say that it is mainly managed by me with some help from Claudio and Jason whereas Core and Project API, I would put both Claudio and Jason as the main maintainers so not confident enough to change the version number haha.

Also, for the project_api version, I guess we can change it but as soon as we change it, all the endpoints will try to look up the project data under that specific version directory. E.g., if the project_api version is 21.5.1, endpoints look for data under a directory called v21p5p1, so either we need to manually rename the project directory every time we update the project API package version or use another method instead of using package version as a projects directory.
Thoughts? @claudio525 @jasonmotha

Reply:
In the future we can make some automated task to run GNZL when we update the API version. In the interim, we shouldn't be doing many changes to the API at the moment - so fine to just deal with it.

Reply:
>Is it also worth to change the core_api version as the core_api endpoints have changed too?

We haven't really been updating these as far as I'm aware, so no good reason to start now (but we should probably start at some point lol)

>Also, for the project_api version, I guess we can change it but as soon as we change it, all the endpoints will try to look up the project data under that specific version directory. E.g., if the project_api version is 21.5.1, endpoints look for data under a directory called v21p5p1, so either we need to manually rename the project directory every time we update the project API package version or use another method instead of using package version as a projects directory.
Thoughts?

I think we should change it so that the directory uses the year-major version, since changes to the project data will always be major. And that way we can do tweaks to the API and just update the minor version number without any problems. But thats probably best done in a separate change. For now, I recon just update the version number and then symlink the GNZL project into the new version directory.

Reply:
Agreed with your point about major version. That's a good distinction.